app/js/ValidationMiddleware.js

```diff
@@ -0,0 +1,9 @@
+const { celebrate } = require('celebrate')
+
+module.exports = { validate }
+
+function validate(schema) {
+  return celebrate(schema, {
+    allowUnknown: true
```

Comment:
This seems handy, but shouldn't we always know which properties it's being called with and when that changes, we change the schema?

Reply:
Right. I hesitated for that one. Because the csrf token is a cross-cutting feature, I'll add it here as an optional param in every schema.